bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/container.jsx

```diff
@@ -9,18 +10,30 @@ import PresentationUploader from './component';
 import { UsersContext } from '/imports/ui/components/components-data/users-context/context';
 import Auth from '/imports/ui/services/auth';
 import { isDownloadPresentationWithAnnotationsEnabled } from '/imports/ui/services/features';
+import { toast } from 'react-toastify';
 
 const PRESENTATION_CONFIG = Meteor.settings.public.presentation;
 
+const unmount = () => {
```

Comment:
I am a bit uncertain about this new `unmount` function that comes from the service... If I am guessing correctily, this would be responsible for correcting the problem described in the PR, right? The one when the user loses the presenter status and the toast doesn't disappear. Could you expand a bit on it?